.github/workflows/dockerhub.yaml

```diff
@@ -2,11 +2,7 @@
 
 name: CD to Docker Hub
 
-on:
-  push:
-    tags:
-      - v[0-9]+.[0-9]+.[0-9]+
-      - v[0-9]+.[0-9]+.[0-9]+-[a-z]+.[0-9]+ # beta/rc releases
+on: workflow_call
```

Comment:
https://docs.github.com/en/actions/learn-github-actions/reusing-workflows
says it should be 
```
on: 
  workflow_call:
```

Reply:
It should be a case when you have a nested parameters there.
This [part of documentation](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#example-2) says that syntax on 1 line might be valid too.

Reply:
We should test it anyway to know for sure.